src/interface/src/app/treatments/after-duplicate-treatment-dialog/after-duplicate-treatment-dialog.component.scss

```diff
@@ -0,0 +1,3 @@
+.padded {
+  padding: 0 16px;
+}
```

Comment:
Should we (or I) somehow include this as part of the dialog component?

Reply:
yeah...i found myself repeating a bunch of stuff just to get a dialogs (like all this special dialog components, which ideally I should be using something more generic).

Reply:
But that would be outside the scope of this pr, i think we need some tlc on wrapping up the modals to be more reusable